src/languages/es.ts

```diff
@@ -4800,4 +4801,19 @@ export default {
         updateRoomDescription: 'establece la descripción de la sala a:',
         clearRoomDescription: 'la descripción de la habitación ha sido borrada',
     },
+    debug: {
+        debug: 'Depuración',
+        details: 'Detalles',
+        JSON: 'JSON',
+        reportActions: 'Actions',
+        reportActionPreview: 'Previa',
+        nothingToPreview: 'Nada que previsualizar',
+        editJson: 'Edit JSON:',
```

Comment:
```suggestion
        editJson: 'Editar JSON:',
```

Reply:
To do it safely, I think we need to confirm the Spanish translation with the design team